native_client/deepspeech.cc

```diff
@@ -170,6 +173,29 @@ struct ModelState {
    */
   char* decode(vector<float>& logits);
 
+  /**
+   * @brief Perform decoding of the logits, using basic CTC decoder or
+   *        CTC decoder with KenLM enabled
+   *
+   * @param logits         Flat matrix of logits, of size:
```

Comment:
"flat matrix" when it is just a vector of floats, can we just say an array ? It might be confusing wording

Reply:
@lissyx This is existing terminology in our code, not added by this PR.

Reply:
another proof that I don't read the doc :+1: